components/psst/browser/content/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+component("content") {
+  output_name = "psst_browser_content"
+
+  defines = [ "IS_PSST_BROWSER_CONTENT_IMPL" ]
```

Comment:
This should be `IS_PSST_IMPL` to match `COMPONENT(PSST)`

Reply:
Why? There's one each for content, core and common

Reply:
because it's not necessary and just makes things more confusing and easier to get wrong